sharpy/linear/assembler/lincontrolsurfacedeflector.py

```diff
@@ -79,6 +79,8 @@ def generate(self, linuvlm=None, tsaero0=None, tsstruct0=None, aero=None, struct
         Returns:
 
         """
+        # In hindsight, building this matrix iterating through structural node was a big mistake that
+        # has led to very messy code. Would rework by element and in B frame
 
         if self.aero is not None:
```

Comment:
If the class has been initialised but it is still given the optional input arguments, it will ignore them. What do you think about changing "if self.aero is not None" by " if aero is None"?
```suggestion
        if aero is None:
```

Reply:
Indeed